classes/class-wc-connect-service-settings-store.php

```diff
@@ -160,5 +160,31 @@ protected function get_service_settings_key( $service_id, $service_instance = fa
 
 			return 'woocommerce_' . $service_id . '_' . $service_instance . '_form_settings';
 		}
+
+		private function translate_unit( $value ) {
+			switch ( $value ) {
+				case 'kg':
+					return __('kg', 'woocommerce');
+				case 'g':
+					return __('g', 'woocommerce');
```

Comment:
@allendav: If we do this, we'll break away from the standard used in WooCommerce itself. I think we should stick with the way it's implemented now, unless this changes in the woocommerce plugin.

```
						<select id="dimension_unit" name="dimension_unit" class="wc-enhanced-select">
							<option value="m" <?php selected( $dimension_unit, 'm' ); ?>><?php echo __( 'm', 'woocommerce' ); ?></option>
							<option value="cm" <?php selected( $dimension_unit, 'cm' ); ?>><?php echo __( 'cm', 'woocommerce' ); ?></option>
							<option value="mm" <?php selected( $dimension_unit, 'mm' ); ?>><?php echo __( 'mm', 'woocommerce' ); ?></option>
							<option value="in" <?php selected( $dimension_unit, 'in' ); ?>><?php echo __( 'in', 'woocommerce' ); ?></option>
							<option value="yd" <?php selected( $dimension_unit, 'yd' ); ?>><?php echo __( 'yd', 'woocommerce' ); ?></option>
						</select>
```